File: Chapter-2/3/quote.py
#! /usr/bin/env python3
#-*- encoding: utf-8
import sys
print("What is the quote? ", end='')
sys.stdout.flush()
quote = sys.stdin.readline().strip()
print("Who said it? ", end='')
sys.stdout.flush()
speaker = sys.stdin.readline().strip()
print("%s says, \"%s\"" % (speaker, quote))

File: Chapter-2/3/quote.rs
use std::io::prelude::*;
use std::io;

fn main() {
  let mut quote = String::new();
  let mut speaker = String::new();

  print!("What is the quote? ");
  io::stdout().flush().unwrap();
  io::stdin().read_line(&mut quote).unwrap();
  quote = quote.trim_right().to_string();

  print!("Who said it? ");
  io::stdout().flush().unwrap();
  io::stdin().read_line(&mut speaker).unwrap();
  speaker = speaker.trim_right().to_string();

  println!("{} says, \"{}\"", speaker, quote);
}
